packages/inferno-compat/README.md

```diff
@@ -39,7 +48,7 @@ Note: Inferno will not currently validate `PropTypes`
 
 ## Usage with Webpack
 
-Using `inferno-compat` with Webpack is easy.
+Using `inferno-compat` with Webpack is easy. You might not always need the `inferno-compat` package. Just the alias below to Inferno is enough.
```

Comment:
This is contradicting. The example below these lines clearly references the inferno-compat package which implies that you do in fact need inferno-compat.

Reply:
Thanks for feedback, I updated inferno-compat readme hopefully its better now! :)